build.gradle

```diff
@@ -64,8 +66,37 @@ dependencies {
     testImplementation("org.bouncycastle:bcpkix-jdk15on:1.68")
 
     testImplementation("ch.qos.logback:logback-classic:1.2.3")
+
+    swaggerCodegen("io.swagger.codegen.v3:swagger-codegen-cli:3.0.24")
+}
+
+swaggerSources {
+    paymentInitiation {
+        inputFile = file('specs/payment-initiation-v3-swagger.yaml')
+        code {
+            language = 'java'
+            components = ['models']
+            configFile = file('specs/payment-initiation-v3-generator-config.json')
+        }
+    }
 }
 
+compileJava.dependsOn(generateSwaggerCode)
+
+sourceSets.main.java.srcDir "${swaggerSources.paymentInitiation.code.outputDir}/src/main/java"
+
+spotless {
+    java {
+        target layout.files("${swaggerSources.paymentInitiation.code.outputDir}/src/main/java")
+        replaceRegex 'Remove swagger @ApiModel annotations', '@ApiModel.*$', ''
```

Comment:
Removing these three annotations means we don't have to include the swagger or javax validation libraries as (api) dependencies of the library. Given the same documentation is present as Javadoc comments in the files, and these files aren't intended to be API request parameters, I don't think there's any need to have swagger documentation on them.